locales/en/index.yml

```diff
@@ -483,6 +483,9 @@ profile:
           title: Do you want to change the language to
           subtitle: You'll need to restart the application to make this happen
         not_selected: Not selected
+      notifications:
+        title: Push notifications
+        subtitle: Consenti anteprima e promemoria
```

Comment:
@thisisjp @AliceAzzolini the English version of this key is missing by purpose or would you like to suggest a translation for the key? 😄

Reply:
@CrisTofani that was not intended, here is the translation:

```suggestion
        subtitle: Allow previews and reminders